modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -146,6 +146,17 @@ The following permissions are not required to create the private {product-title}
 * `Microsoft.Storage/storageAccounts/write`
 ====
 
+.Required permissions for creating an image registry storage account private endpoint
```

Comment:
The required permission in this doc is that it is always required when you deploying IPI cluster whatever the configuration. But here it is optional operation to create an image registry storage account private endpoints and only exist on private cluster, so suggest to change `Optional permissions`

Reply:
Renamed ".Optional permissions for creating an image registry storage account private endpoint"